src/hooks/useLoadDocuments.ts

```diff
@@ -1,31 +1,34 @@
 import useLoadFromEndpoint, { ReturnType } from './useLoadFromEndpoint'
-import { AlternativeWordType, DocumentType, ENDPOINTS } from '../constants/endpoints'
+import { DocumentType, ENDPOINTS } from '../constants/endpoints'
 import { ARTICLES } from '../constants/data'
 
+export interface AlternativeWordTypeFromServer {
+  article: number
+  alt_word: string
+}
+
 export interface DocumentTypeFromServer {
   id: number
   word: string
   article: number
   document_image: Array<{ id: number; image: string }>
   audio: string
-  alternatives: AlternativeWordType[]
+  alternatives: AlternativeWordTypeFromServer[]
 }
 
 const transformArticle = (documents: ReturnType<DocumentTypeFromServer[]>): ReturnType<DocumentType[]> => {
-  const documentsWithTransformedArticle: DocumentType[] = []
-  documents.data?.forEach(doc => {
-    const { article, ...rest } = doc
-    const formattedArticle = ARTICLES.find(article => {
-      return article.id === doc.article
-    })
-    if (!formattedArticle) {
-      throw new Error('Invalid Server Response')
-    }
-    const formattedDocument: DocumentType = { ...rest, article: formattedArticle }
-    documentsWithTransformedArticle.push(formattedDocument)
-  })
-  const { data, ...rest } = documents
-  return { ...rest, data: documentsWithTransformedArticle }
+  let formattedDocuments: DocumentType[] = []
+  if (documents.data) {
+    formattedDocuments = documents.data?.map(document => ({
+      ...document,
+      article: ARTICLES[document.article],
+      alternatives: document.alternatives.map(it => ({
+        article: ARTICLES[it.article],
+        word: it.alt_word
+      }))
+    }))
+  }
```

Comment:
```suggestion
  const formattedDocuments: DocumentType[] = documents.data?.map(document => ({
      ...document,
      article: ARTICLES[document.article],
      alternatives: document.alternatives.map(it => ({
        article: ARTICLES[it.article],
        word: it.alt_word
      }))
    })) ?? []
```
should work as well.